templates/test/core/surface/public_headers_must_be_c89.c.template

```diff
@@ -35,9 +35,23 @@
       assert(hdr[0:len(pfx)] == pfx)
       hdrs.add(hdr[len(pfx):])
   hdrs = sorted(list(hdrs))
+  fns = list()
+  for api in c_apis:
+    if is_platform_header(api.header):
+      continue
+    fns.append(api.name)
   %>\
   % for hdr in hdrs:
   #include <${hdr}>
   % endfor
 
-  int main(int argc, char **argv) { return 0; }
+  #include <stdio.h>
+
+  int main(int argc, char **argv) {
+    if(argc == 12345678) {
```

Comment:
nit: if we're only printing the functions, we don't need the if (argc...) business